.github/workflows/pr-review.yml

```diff
@@ -0,0 +1,43 @@
+name: PR Preview
+on:
+  pull_request:
+    types: [synchronize]
+
+jobs:
+  Release-Snapshot:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          version: 14.x
+      - name: Install Dependencies
+        run: yarn
+      - name: Exit prerelease mode if on
+        run: |
+          [ -f .changeset/pre.json ] && yarn changeset pre exit
+      - name: Run changeset version
+        run: yarn changeset version --snapshot pr-${{ github.event.number }}
+      - name: Log package version changes
+        run: cat ${{ github.workspace }}/packages/core/package.json
+      - name: Creating .npmrc
+        run: |
+          cat << EOF > "$HOME/.npmrc"
+            email=wkl.hyo@gmail.com
+            //registry.npmjs.org/:_authToken=$NPM_TOKEN
+          EOF
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Publish to pr-# tag
+        run: yarn changeset publish --tag pr-${{ github.event.number }}
```

Comment:
This tag name can be unique for each PR like so (PR-#), or we can use a same tag name across all PRs (this is what auto does).